server/chat-plugins/hosts.ts

```diff
@@ -0,0 +1,352 @@
+/**
+ * Chat plugin to help manage hosts, proxies, and datacenters
+ * Written by Annika
+ * Original /adddatacenters command written by Zarel
+ */
+
+import {Utils} from "../../lib/utils";
+import {Datacenter} from "../ip-tools";
+
+const IP_REGEX = /^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$/;
```

Comment:
I think I've seen this regex in a couple CLs now. Would it make sense for it to be a constant on Utils if its used in multiple places?

Reply:
Based on my searching, it's only really used here -- there's an `IPS_REGEX` in modlog, but that searches for IPs specifically in the context of modlogs (i.e. with `[a.b.c.d]` format), so I'm not sure it should be in Utils.

Reply:
(I might be missing a place, though)